### PR TITLE
add jyg 0.1.1

### PR DESCRIPTION
**Pull request by bollwyvl**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [ ] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [ ] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [ ] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by bollwyvl** [comment]
@conda-forge/help-python @conda-forge/staged-recipes ready for review

File: recipes/jyg/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+    - jupyter server extension list 1>server_extensions 2>&1
+    - grep -iE "@deathbeds/jyg.*OK" labextensions
+    - grep -iE "jyg.*OK" server_extensions
+  requires:
+    - jupyterlab
+    - m2-grep  # [win]
+    - pip
+
+about:
+  home: https://github.com/sathbeds/jyg
```
Comment (minrk) [inline recipes/jyg/meta.yaml:49]: typo:

```suggestion
  home: https://github.com/deathbeds/jyg
```

File: recipes/jyg/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+    - python -m jyg --help
+    - jupyter labextension list
+    - jupyter server extension list
+    - jupyter labextension list 1>labextensions 2>&1
+    - jupyter server extension list 1>server_extensions 2>&1
+    - grep -iE "@deathbeds/jyg.*OK" labextensions
+    - grep -iE "jyg.*OK" server_extensions
+  requires:
+    - jupyterlab
+    - m2-grep  # [win]
```
Comment (minrk) [inline recipes/jyg/meta.yaml:45]: To handle this with noarch, I think you need `__unix # [not win]` and `__win # [win]` in runtime deps, right? That will prevent an incorrect package being published for Windows until you can add the necessary conda-forge.yml [as we did here](https://github.com/conda-forge/ipykernel-feedstock/pull/127/files#diff-478786365dd93f740eb520c2faa03d7a0623273663b75472272c8ef94297bbe2R9-R12)

File: recipes/jyg/meta.yaml (hunk 1 above)
Comment (bollwyvl) [reply recipes/jyg/meta.yaml:49]: doh thanks!

File: recipes/jyg/meta.yaml (hunk 2 above)
Comment (bollwyvl) [reply recipes/jyg/meta.yaml:45]: i have never run into issues with _test_ dependencies having selectors... i can do some looking...

**Pull request merged**